### PR TITLE
Persitence layer

### PR DESCRIPTION
**Pull request by washington1983**
We have changed some methods in actor service to be used.

### CODE REVIEW

File: src/main/java/upm/softwaredesign/finalproject/service/ActorService.java [hunk 1]
```diff
@@ -2,83 +2,96 @@
...
+		for(ActorEntity actorEntity : actorsEntityList) {
+			actors.add(cast(actorEntity));
 		}
-    	return ;
-    }
-	*/
+		return actors;
+	}
+
+	public List<Actor> retrieveActorBytype(String actorType) {
```
Comment (kaca1707) [inline src/main/java/upm/softwaredesign/finalproject/service/ActorService.java]: change to retrieveActorByType

File: src/main/java/upm/softwaredesign/finalproject/service/ActorService.java [hunk 2]
```diff
@@ -2,83 +2,96 @@
...
+		// TODO: We need to have a logic to distinguish type of actor and create
+		// Appropriate type
 		Actor actor = new Retailer();
 		actor.setName(actorEntity.getName());
 		actor.setType(actorEntity.getType());
 		return actor;
 	}
-	public Actor findByType(String type ) {
+
+	public Actor findByType(String type) {
```
Comment (kaca1707) [inline src/main/java/upm/softwaredesign/finalproject/service/ActorService.java]: Is this method needed? There is now retrieveActorByType

**Review by kaca1707 — CHANGES_REQUESTED** [review]

File: src/main/java/upm/softwaredesign/finalproject/service/ActorService.java (hunk 1 above)
Comment (washington1983) [reply src/main/java/upm/softwaredesign/finalproject/service/ActorService.java]: Done

File: src/main/java/upm/softwaredesign/finalproject/service/ActorService.java (hunk 2 above)
Comment (washington1983) [reply src/main/java/upm/softwaredesign/finalproject/service/ActorService.java]: It was deleted

**Review by kaca1707 — APPROVED** [review]

**Comment by izzet** [comment]
I have implemented test methods as well, they are included in this PR.

**Review by kaca1707 — APPROVED** [review]

**Review by piotrsiwinski — APPROVED** [review]
Hello, looks good to me ;) i think that we can merge

**Pull request merged**